Codeinwp/neve
# Change the way we detect if shop or product has elementor template

### Summary
* Improvement on the Elementor template detecting mechanism (Leveraging as soon as possible from Elementor code base)
* Function parameters of the Elementor::is_elementor_template has been changed. (from now on, one param is passed. (available params also has been changed.))

### Will affect the visual aspect of the product
NO

### Test instructions
**See: https://github.com/Codeinwp/neve-pro-addon/issues/2116#issuecomment-1304877130**
Please test with https://github.com/Codeinwp/neve-pro-addon/pull/2150

**Cases**
- See the issue text and make sure the issue has been fixed.
- Make sure there is no fatal error in case of usage [Neve this PR + stable Neve Pro] or [stable Neve + https://github.com/Codeinwp/neve-pro-addon/pull/2150]


Closes Codeinwp/neve-pro-addon#2116.
